### PR TITLE
Show user nick if available

### PR DESCRIPTION
**Pull request by hharchani**

### CODE REVIEW

File: src/app/views/skeleton_template/header.php [hunk 1]
```diff
@@ -45,7 +45,16 @@
...
         <?php if ($is_authenticated): ?>
-            <div><a href="<?= base_url() . "auth/logout/" ?>" class="pure-button btn">Logout</a></div>
+            <div>
+                <?php if (isset($user_nick) && $user_nick): ?>
+                    <div class="nick">
+                        Hello <?= $user_nick ?>,
+                        <a href="<?= base_url() . "auth/logout/" ?>">Logout</a>
+                    </div>
+                <?php else: ?>
+                    <a href="<?= base_url() . "auth/logout/" ?>" class="pure-button btn">Logout</a>
```
Comment (nisargjhaveri) [inline src/app/views/skeleton_template/header.php:55]: We can have a link to complete registration here.

File: src/app/libraries/ldap_lib.php [hunk 2]
```diff
@@ -7,7 +7,6 @@ class ldap_lib extends Library {
 
     public function get_link($value='') {
         global $ldap_cfg;
-
```
Comment (nisargjhaveri) [inline src/app/libraries/ldap_lib.php:10]: Why this change? :P

File: src/app/views/skeleton_template/header.php (hunk 1 above)
Comment (hharchani) [inline src/app/views/skeleton_template/header.php:55]: Not now.

Comment (hharchani) [inline src/app/views/skeleton_template/header.php:55]: oh wait, complete registration. Sure. what is the link?

Comment (nisargjhaveri) [inline src/app/views/skeleton_template/header.php:55]: `/auth/register/`

Comment (nisargjhaveri) [inline src/app/views/skeleton_template/header.php:55]: Also, why two different styles for the same button?

Comment (hharchani) [inline src/app/views/skeleton_template/header.php:55]: Because if the nick is there, I've to show it as a link, otherwise as a button.

Comment (hharchani) [inline src/app/views/skeleton_template/header.php:55]: How do I detect if user haven't completed the registration?

Comment (nisargjhaveri) [inline src/app/views/skeleton_template/header.php:55]: Oh, you can't do it easily.

File: src/app/libraries/ldap_lib.php (hunk 2 above)
Comment (hharchani) [inline src/app/libraries/ldap_lib.php:10]: Galti se :/

**Comment by polybuildr** [comment]
Why the close?
